Application/EdFi.Ods.Standard/Artifacts/PgSql/Structure/Ods/1150-UpdatedAuthViewsToPreventDuplicatesFromStaffEdOrgAssignmentsAndEmployments.sql

```diff
@@ -0,0 +1,118 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+-- Drop dependent views
+DROP VIEW auth.EducationOrganizationIdToStaffUSI;
```

Comment:
should there be an IF EXISTS condition here?

Reply:
Probably wouldn't hurt to add it, but realistically the view would always be there when this script runs and then it gets recreated at the end of this script, so unless the script fails halfway through, it would still be idempotent.